README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Kieran Coffey
+I am a MAPSS student studying European History - specifically German cultural and political history of the nineteenth and twentieth centuries. At the moment I am studying the relationship between conceptions of regional, national, and religious identity as they relate to the German concept of *Heimat*, or homeland. My other academic interests include:
+
+    - nationalism and national identity
```

Comment:
nice use of a list here! it's cool that the font and the background changed. Not sure how you did that.

Reply:
It's actually not a correctly processed Markdown list. Don't use indents when initializing a list. Compare this

```
    - nationalism and national identity
    - sub-national or regional identity
    - mass political movements, particularly socialist/communist and far-right movements
    - borderland regions, particular between Germany and France and the European Union 
```

    - nationalism and national identity
    - sub-national or regional identity
    - mass political movements, particularly socialist/communist and far-right movements
    - borderland regions, particular between Germany and France and the European Union 

With this:

```
- nationalism and national identity
- sub-national or regional identity
- mass political movements, particularly socialist/communist and far-right movements
- borderland regions, particular between Germany and France and the European Union 
```

- nationalism and national identity
- sub-national or regional identity
- mass political movements, particularly socialist/communist and far-right movements
- borderland regions, particular between Germany and France and the European Union